app/newRoot.css

```diff
@@ -51,6 +51,10 @@
   flex-grow: 2;
 }
 
+.verticaly-centered {
```

Comment:
vertically* is the correct English spelling, not sure if that matters though lol